util/src/main/i18n/templates/gamemode.properties

```diff
@@ -211,6 +211,11 @@ flag.return = {0} has been returned
 # {1} = duration (e.g. "5 seconds")
 flag.willRespawn = {0} will respawn in {1}
 
+# {0} = flag name
+# {1} = post name
+# {2} = duration
+flag.willRespawn.named = {0} will respawn to {1} in {2}
```

Comment:
Also, should this be `{0} will respawn at {1} in {2}` ?

Reply:
I think so; I will change it